DEVDOCS-2670: Update Config Meta Usage (#1085)
bigcommerce/dev-docs: docs/api-docs/channels/channel-app-requirements.md

All Partners:

> - Uses [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)

Reviewer: ```suggestion
- Uses [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api).

```

- [Creates a channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel) upon app installation.

> - Reads and updates channel status via [Channels endpoints](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)

Reviewer: ```suggestion
- Reads and updates channel status using the [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api) endpoints

```

Select Partners:

> - Uses [Big Design](https://developer.bigcommerce.com/big-design/)

Reviewer: ```suggestion
- Uses [Big Design](https://developer.bigcommerce.com/big-design/).

```
